config/default/kustomization.yaml

```diff
@@ -21,7 +21,7 @@ bases:
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
 #- ../certmanager # Comment for KCP
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
-#- ../prometheus
+- ../prometheus
```

Comment:
What happens if Prometheus isn't installed on the Kubernetes cluster when you do `make deploy`?

Reply:
It doesn't break but I should comment it out since it install a ServiceMonitor that doesn't get used.  We actually define the SM on the infra-deploy repo